### PR TITLE
Feature/graphql promos deletion

### PR DESCRIPTION
**Pull request by lafbelmonte**
-graphql mutation for deleting a promo

### CODE REVIEW

File: koa-crud/src/graphql/typeDefs/promos.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ export default gql`
   extend type Mutation {
     createPromo(input: CreatePromoInput!): Boolean
     updatePromo(input: UpdatePromoInput!): Boolean
+    deletePromo(id: ID!): Boolean
```
Comment (ralphcasipe1) [inline koa-crud/src/graphql/typeDefs/promos.ts]: make the return required

**Review by ralphcasipe1 — CHANGES_REQUESTED** [review]
Add `!` on the return of your promo's mutation

File: koa-crud/src/graphql/typeDefs/promos.ts (hunk 1 above)
Comment (lafbelmonte) [reply koa-crud/src/graphql/typeDefs/promos.ts]: noted on this sir

**Review by ralphcasipe1 — APPROVED** [review]
:100:  LGTM!

**Pull request merged**